File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/verification/pharmacy/shared/pharmacy-requisition-verification.dto.ts
export class PharmacyRequisitionVerification_DTO {
    RequisitionId: number;
    RequisitionNo: number;
    RequisitionDate: string;
    RequisitionStatus: string;
    RequestedBy: string;
    DispatchedBy: string;
    ReceivedBy: string;
    RequestedStoreName: string;
    public CancelledBy: number = null;
    public CancelledOn: string = "";
    public CancelRemarks: string = "";
    CurrentVerificationLevel: number = 0;
    MaxVerificationLevel: number = 0;
    CurrentVerificationLevelCount: number = 0;
    VerificationStatus: string = null;
    IsVerificationAllowed: boolean = false;
    VerificationId: number = 0;
    VerifierIds: string = null;
    IsVerificationEnabled: boolean = false;
}

File: Code/Websites/DanpheEMR/Services/Verification/DTOs/PharmacyRequisitionVerification_DTO.cs
using System;

namespace DanpheEMR.Services.Verification.DTOs
{
    public class PharmacyRequisitionVerification_DTO
    {
        public int RequisitionId { get; set; }
        public int RequisitionNo { get; set; }
        public DateTime RequisitionDate { get; set; }
        public string RequisitionStatus { get; set; }
        public string RequestedBy { get; set; }
        public string RequestedStoreName { get; set; }
        public int? CancelledBy { get; set; }
        public string CancelledOn { get; set; }
        public string CancelRemarks { get; set; }
        public int CurrentVerificationLevel { get; set; }
        public int MaxVerificationLevel { get; set; }
        public int CurrentVerificationLevelCount { get; set; }
        public string VerificationStatus { get; set; }
        public bool IsVerificationAllowed { get; set; }
        public int? VerificationId { get; set; }
        public string VerifierIds { get; set; }
        public bool IsVerificationEnabled { get; set; }
    }
}
